src/test/java/jenkins/metrics/api/ConfigAsCodeTest.java

```diff
@@ -0,0 +1,80 @@
+package jenkins.metrics.api;
+
+import hudson.ExtensionList;
+import io.jenkins.plugins.casc.ConfigurationAsCode;
+import io.jenkins.plugins.casc.ConfigurationContext;
+import io.jenkins.plugins.casc.Configurator;
+import io.jenkins.plugins.casc.ConfiguratorRegistry;
+import io.jenkins.plugins.casc.model.CNode;
+import io.jenkins.plugins.casc.model.Mapping;
+import java.util.List;
+import jenkins.metrics.api.MetricsAccessKey.DescriptorImpl;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.junit.Assert.assertNotNull;
+
+public class ConfigAsCodeTest {
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
+
+    @Test
+    public void should_support_jcasc_from_yaml() throws Exception {
+        ConfigurationAsCode
```

Comment:
use ConfiguredWithCode annotation / rule?
e.g.
https://github.com/jenkinsci/azure-keyvault-plugin/pull/15/files

Reply:
I cannot it is not in packaged

Reply:
seems to work in that PR, you need to add another dep with the tests classifier

Reply:
Or at least I thought so